src/components/Modals/TagsAndCategories/CreateCategoryDialog.vue

```diff
@@ -69,14 +69,14 @@ export default {
   },
   data: () => ({
     nameRules: [
-      v => !!v || 'Category name is required',
+      v => !!v || this.$i18n.t('modals.newCategory.tipOnNoName'),
```

Comment:
This doesn't seem to work 😞

Reply:
found a way to fix it though

Reply:
> found a way to fix it though

I don't have Vue relevant skills. The frontend is too difficult!

Reply:
No worries! It has become quite complex in some areas, I don't blame you 😅